### ISSUE TITLE
the side &deep model   is not good compare with deep model and wide model 

### ISSUE THREAD

**Issue by lijhong** [tensorflow/tensorflow#12764]
these days ,I'm learning the wide & deep model ,and run the  wide_n_deep_tutorial.py, so the anwser looks like this:

XXT@apptruexxnet:~$ python wide_n_deep_tutorial.py  --model_type=deep
Training data is downloaded to /tmp/tmpFB4dsd
Test data is downloaded to /tmp/tmpomj5Pi
2017-09-02 20:03:15.713609: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.1 instructions, but these are available on
 your machine and could speed up CPU computations.
2017-09-02 20:03:15.713884: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on
 your machine and could speed up CPU computations.
2017-09-02 20:03:15.714050: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on yo
ur machine and could speed up CPU computations.
WARNING:tensorflow:Casting <dtype: 'float32'> labels to bool.
WARNING:tensorflow:Casting <dtype: 'float32'> labels to bool.
model directory = /tmp/tmpWei9wK
accuracy: 0.850071
accuracy_baseline: 0.763774
auc: 0.894038
auc_precision_recall: 0.743199
average_loss: 0.393638
global_step: 2000
label/mean: 0.236226
loss: 39.3179
prediction/mean: 0.242167

XXT@apptruexxnet:~$ python wide_n_deep_tutorial.py  --model_type=wide
Training data is downloaded to /tmp/tmpFJdWft
Test data is downloaded to /tmp/tmpjB5nm7
2017-09-02 20:01:09.197612: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.1 instructions, but these are available on
 your machine and could speed up CPU computations.
2017-09-02 20:01:09.197906: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on
 your machine and could speed up CPU computations.
2017-09-02 20:01:09.198072: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on yo
ur machine and could speed up CPU computations.
WARNING:tensorflow:Casting <dtype: 'float32'> labels to bool.
WARNING:tensorflow:Casting <dtype: 'float32'> labels to bool.
model directory = /tmp/tmpuPHsDx
accuracy: 0.835391
accuracy_baseline: 0.763774
auc: 0.882763
auc_precision_recall: 0.694257
average_loss: 0.352975
global_step: 2000
label/mean: 0.236226
loss: 35.2563
prediction/mean: 0.240918

XXT@apptruexxnet:~$ python wide_n_deep_tutorial.py
Training data is downloaded to /tmp/tmpDdWc_T
Test data is downloaded to /tmp/tmpFF0PZJ
2017-09-02 20:00:08.334742: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.1 instructions, but these are available on
 your machine and could speed up CPU computations.
2017-09-02 20:00:08.335105: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on
 your machine and could speed up CPU computations.
2017-09-02 20:00:08.335273: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on yo
ur machine and could speed up CPU computations.
WARNING:tensorflow:Casting <dtype: 'float32'> labels to bool.
WARNING:tensorflow:Casting <dtype: 'float32'> labels to bool.
model directory = /tmp/tmpq2M7SE
accuracy: 0.820834
accuracy_baseline: 0.763774
auc: 0.850518
auc_precision_recall: 0.676198
average_loss: 0.424271
global_step: 2000
label/mean: 0.236226
loss: 42.3776
prediction/mean: 0.256489

I don't know why looks likes this, so someone can help me? thank you.

**Comment by carlthome** [comment]
This is not a bug?

**Comment by michaelisard** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!

**Comment by lijhong** [comment]
alright,thanks.
